dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/data/upload/UploadScheduler.kt

```diff
@@ -0,0 +1,6 @@
+package com.datadog.android.core.internal.data.upload
+
+internal interface UploadScheduler {
+    fun startScheduling()
```

Comment:
just out of consistency, i'd call the methods `start`/ `stop` or `startScheduling`/ `stopScheduling`